tests/lib/rules/jsx-curly-spacing.js

```diff
@@ -45,13 +60,21 @@ ruleTester.run('jsx-curly-spacing', rule, {
     options: ['never'],
     parserOptions: parserOptions
   }, {
+    code: [
+      '<App foo={',
+      '{ bar: true, baz: true }',
+      '} />;'
+    ].join('\n'),
+    options: ['never'],
+    parserOptions: parserOptions
+  }, {
     code: '<App foo={ {bar:baz} } />;',
     options: ['always'],
     parserOptions: parserOptions
   }, {
     code: [
       '<App foo={',
-      'bar',
```

Comment:
This test is identical with the one just below it, so I suggest it stays removed.